internal/manager/setup.go

```diff
@@ -287,7 +288,7 @@ func (c *Config) adminAPIClients(ctx context.Context) ([]*adminapi.Client, error
 	return clients, nil
 }
 
-func (c *Config) adminAPIClientFromServiceDiscovery(ctx context.Context, httpclient *http.Client) ([]*adminapi.Client, error) {
+func (c *Config) adminAPIClientFromServiceDiscovery(ctx context.Context, httpclient *http.Client, adminSvc types.NamespacedName) ([]*adminapi.Client, error) {
```

Comment:
I'd personally leave it as it was and change where needed in the func definition.